inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -15,11 +15,15 @@
 from inductiva.simulation import Simulator
 from inductiva.fluids.simulators import OpenFOAM
 from inductiva.utils.templates import (TEMPLATES_PATH,
-                                       batch_replace_params_in_template)
+                                       batch_replace_params_in_template,
+                                       replace_params_in_template)
 from inductiva.utils.files import remove_files_with_tag
+from inductiva.fluids.scenarios.wind_tunnel.post_processing import WindTunnelSimulationOutput
 
 SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "wind_tunnel")
 OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam"
+FILES_SUBDIR = "files"
```

Comment:
For curiosity purpose only, what are the files under `files` dir?

Reply:
Are the actual template files to configure the simulation. This folder structure simplifies the work flow with the commands files. In this way, they are not passed to the input_dir.